src/bio2bel/sources/biogrid.py

```diff
@@ -0,0 +1,268 @@
+# -*- coding: utf-8 -*-
+
+"""This script downloads and parses BioGRID data and maps the interaction types to BEL.
+
+To run this script, install Bio2BEL and then do:
+
+python -m bio2bel.sources.biogrid
```

Comment:
Please add description (written as english prose, not technical documentation) about what's in BioGRID (what kinds of relations), what the format is, and what the process was like to convert it (had to map MI relations to BEL PTMs, gene modifications, associations, etc.)

Reply:
I will write a bit about how one goes about dealing with databases that have errant identifiers